install/generator/01-header.yml.mustache

```diff
@@ -30,6 +30,10 @@ parameters:
   description: The name of the OpenShift project Syndesis is being deployed into.
   displayName: OpenShift project to deploy into
   required: true
+- name: SAR_PROJECT
```

Comment:
Not sure whether we want to have this for 1.4.x, too. This is another feature which requires also an update of the installation scripts (i.e. that template parameter is mandatory and should default to the OpenShift project). 

We should not add this to 7.1.